x-pack/plugins/ml/public/application/datavisualizer/index_based/components/search_panel/field_name_filter.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { FC, useEffect, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { Option, MultiSelectPicker } from '../../../../components/multi_select_picker';
+import type { OverallStats } from '../../../../../../common/types/datavisualizer';
+
+interface Props {
+  overallStats: OverallStats;
+  setVisibleFieldNames(q: any): void;
```

Comment:
is it an array of strings?

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/4ea6d174728033f29d81392da47db1f3b409d4a5